client/client.go

```diff
@@ -21,6 +21,7 @@ type Client interface {
 	GetOffChainData(ctx context.Context, hash common.Hash) ([]byte, error)
 	ListOffChainData(ctx context.Context, hashes []common.Hash) (map[common.Hash][]byte, error)
 	SignSequence(ctx context.Context, signedSequence types.SignedSequence) ([]byte, error)
+	SignSequenceBanana(ctx context.Context, signedSequence types.SignedSequenceBanana) ([]byte, error)
```

Comment:
Is this very specific for Banana or it will be also for future forks? Looks like a very specific method name?

Reply:
who knows 🙃 it starts on banana, and *could* keep working on following versions or not, depends on future protocol updates